src/js/rx/components/TabNav.jsx

```diff
@@ -7,7 +7,7 @@ class TabNav extends React.Component {
     return (
       <ul className="va-tabs rx-nav va-dnp">
         <TabItem tabpath="/rx" title="Refill Active Prescriptions"/>
-        <TabItem tabpath="/rx/history" title="View Prescription History"/>
+        <TabItem tabpath="history" title="View Prescription History"/>
```

Comment:
I think you should keep this. My console gives me this error when I click on the History tab:
```
Warning: [react-router] Location "history" did not match any routes
```